src/03-ticket-calculator.js

```diff
@@ -54,7 +55,34 @@ const exampleTicketData = require("../data/tickets");
     calculateTicketPrice(tickets, ticketInfo);
     //> "Entrant type 'kid' cannot be found."
  */
-function calculateTicketPrice(ticketData, ticketInfo) {}
+function calculateTicketPrice(ticketData, ticketInfo) {
+  let ticketPrice = 0
+  // return meddage if ticket type is not found run false
+  if (!ticketData[ticketInfo.ticketType]) {
+    return `Ticket type '${ticketInfo.ticketType}' cannot be found.`;
+  }
+  // return message if if ticket entrant type does not exist in the price in cennts
+  if (!ticketData[ticketInfo.ticketType].priceInCents[ticketInfo.entrantType]) {
+    return "Entrant type 'incorrect-entrant' cannot be found."
```

Comment:
This is considered hardcoded. If the entrantType was `kid`, we would still want this message with the incorrect entrant type in the string.